src/run_tribler.py

```diff
@@ -94,6 +95,10 @@ def init_boot_logger():
         from tribler.core.components.reporter.exception_handler import default_core_exception_handler
 
         init_sentry_reporter(default_core_exception_handler.sentry_reporter)
+
+        if os.environ.get('SLOW_CORO_STACK_TRACING') == '1':
```

Comment:
Perhaps it is handier to add an option to always trace a stack for developers.
Something like:
```suggestion
        if os.environ.get('SLOW_CORO_STACK_TRACING') == '1' or is_run_from_source():
```

Reply:
It is possible, but I'm not sure we really want this, as it slows doun the code and it may be harder to turn the option off when running from sources

Reply:
As a developer, I really want this.

Reply:
To me, it seems like a good strategy to enable all debugging tools on each developer's machine and use environment variables as an option to disable a tool.

For example, if you're developing 3 tools, I'm developing 2 tools, and Sandip is developing 3 tools, that means Marcel would need to:
1. Know about all 8 tools
1. Enable all of them by setting 8 environment variables

Reply:
Agreed. I changed the code to turn on stack tracking when running from sources by default and allow easier overriding
```python
slow_coro_stack_tracking = os.environ.get('SLOW_CORO_STACK_TRACING', '0' if is_frozen() else '1')
# By default, the stack tracking of slow coroutines is enabled when running the Tribler from sources
# and disabled in the compiled version, as it makes the Python code of Core work slower.
if slow_coro_stack_tracking == '1':
    start_main_thread_stack_tracing()
```

Reply:
Thank you 🙏